app/helpers/application_helper/button/new_host_aggregate.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationHelper::Button::NewHostAggregate < ApplicationHelper::Button::ButtonNewDiscover
+  def supports_button_action?
+    @supported_button_action ||= begin
```

Comment:
Why do you need this?

Reply:
yes, it is not needed, I am not calling it twice now.